### PR TITLE
CLOUD- update EAP 7.1 build config for 7.1.6

### PR DESCRIPTION
**Pull request by luck3y**
https://issues.jboss.org/browse/CLOUD-3090
Signed-off-by: Ken Wills <kwills@redhat.com>
Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[CLOUD-XYA] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: dev-overrides.yaml [hunk 1]
```diff
@@ -12,8 +12,8 @@ modules:
                   ref: master
           - name: jboss-eap-7-image
             git:
-￼                   url: https://github.com/jboss-container-images/jboss-eap-7-image.git
-￼                   ref: eap71-dev
+                  url: https://github.com/jboss-container-images/jboss-eap-7-image.git
+                  ref: eap71-dev
```
Comment (apintea) [inline dev-overrides.yaml:17]: A spacing change would be rejected in other cases, but as you're the maintainer, that's fine :P

**Review by apintea — COMMENTED** [review]
The changes look good to me. I'm totally not an expert at this, but the logic for the updates seems reasonable. Thank you!

File: dev-overrides.yaml (hunk 1 above)
Comment (luck3y) [reply dev-overrides.yaml:17]: Thanks, will fix.

Comment (luck3y) [reply dev-overrides.yaml:17]: Oh, actually this is correcting WS, so I'm ok with leaving it.

**Review by bstansberry — APPROVED** [review]

**Pull request merged**